### PR TITLE
Pass refId as string

### PR DESCRIPTION
**Pull request by kaisermann**
#### What does this PR do? \*

Continuation of #16. It replaces the object passed with the string value.

#### How to test it? \*

See the steps in #16

### CODE REVIEW

File: react/.eslintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "vtex-react",
+  "extends": "vtex-react/io",
```
Comment (emersonlaurentino) [inline react/.eslintrc]: what?

Comment (kaisermann) [reply react/.eslintrc]: Preset exclusively for io apps: https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex-react/io.js

The rationale behind it is to have separate rules for react native and io apps.

Comment (emersonlaurentino) [reply react/.eslintrc]: who uses our rules in react native applications?

Comment (kaisermann) [reply react/.eslintrc]: Possibly instore related projects, but you're free to use the `vtex-react` one as well.

Comment (emersonlaurentino) [reply react/.eslintrc]: strange

**Pull request merged**